### ISSUE TITLE
GKSQT income network connections dialog (MacOS)

### ISSUE THREAD

**Issue by hendri54** [jheinen/GR.jl#412]
When I plot with the GR backend (using Plots.jl or using GR.jl directly), a window pops up that asks:

Do you want the application “gksqt.app” to accept incoming network connections?

This happens for every plot. Is there a way to avoid this? A MWE example is as simple as

```
using GR
histogram(rand(1000))
```

I am on GR 0.57.4 and MacOS:

```
julia> versioninfo()
Julia Version 1.6.1
Commit 6aaedecc44 (2021-04-23 05:59 UTC)
Platform Info:
  OS: macOS (x86_64-apple-darwin18.7.0)
  CPU: Intel(R) Core(TM) i9-9880H CPU @ 2.30GHz
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-11.0.1 (ORCJIT, skylake)
Environment:
  JULIA_NUM_THREADS = 8
```

**Comment by jheinen** [comment]
This problem is already solved in the development branch and will thus be fixed with the next release (0.58.0)

**Comment by hendri54** [comment]
I appreciate it.
